### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 6 - Esercizio 7

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#575] [Esercizi M-Z]
![es7](https://github.com/user-attachments/assets/e9f1c9cb-ba0c-4da4-9b52-bd064748cc74)

**Comment by alem1105** [comment]
Spero sia giusto 🥹🥹

![image](https://github.com/user-attachments/assets/ab162219-2599-4b2c-a048-091e81c87608)
